Library/src/Systems/Organizations/OrganizationSystem.cpp

```diff
@@ -262,21 +375,14 @@ void OrganizationSystem::GetUserRolesInOrganization(const csp::common::Optional<
 	}
 	else
 	{
-		const csp::common::Array<csp::common::String>& CurrentOrganizationIds = GetCurrentOrganizationIds();
+		SelectedOrganizationId = GetCurrentOrganizationId();
 
-		if (CurrentOrganizationIds.Size() == 0)
+		if (SelectedOrganizationId.IsEmpty())
 		{
-			CSP_LOG_ERROR_MSG("OrganizationSystem::InviteToOrganization failed: You do not belong to an Organization.");
```

Comment:
Same question here about removing the error log message. It seemed useful!

Reply:
Not sure. Agreed they are useful so I have added error logs for all of these.